askama_derive/src/generator.rs

```diff
@@ -345,10 +345,9 @@ impl<'a> Generator<'a> {
                     self.flush_ws(m.ws1);
                     self.prepare_ws(m.ws2);
                 }
-                Node::Raw(ws1, contents, ws2) => {
+                Node::Raw(ws1, contents, _) => {
                     self.handle_ws(ws1);
                     self.buf_writable.push(Writable::Lit(contents));
-                    self.handle_ws(ws2);
```

Comment:
This should probably be `self.prepare_ws(ws2);` instead of being removed completely?